packages/components/src/horizontal-bar-list/stats-card.scss

```diff
@@ -117,6 +117,7 @@
 		}
 
 		.stats-card__content .stats-card--header-and-body .stats-card--body,
+		.stats-card__content .stats-card-header--split .stats-card--column-header,
```

Comment:
This is to blur the column headers when splitHeader is passed.

| Before | After |
|---|---|
| ![CleanShot 2025-01-07 at 13 05 19@2x](https://github.com/user-attachments/assets/64100b33-df81-4eac-a833-d394295c9bea) | ![CleanShot 2025-01-07 at 13 06 04@2x](https://github.com/user-attachments/assets/87978585-5268-4a0b-9e87-d98319c816f8) |